infra/prometheus/prometheus.yml

```diff
@@ -0,0 +1,41 @@
+global:
+  scrape_interval: 15s
+  scrape_timeout: 10s
+  scrape_protocols:
+    - OpenMetricsText1.0.0
+    - OpenMetricsText0.0.1
+    - PrometheusText0.0.4
+  evaluation_interval: 15s
+alerting:
+  alertmanagers:
+    - follow_redirects: true
+      enable_http2: true
+      scheme: http
+      timeout: 10s
+      api_version: v2
+      static_configs:
+        - targets: [ ]
+scrape_configs:
```

Comment:
Because this is designed to be used with cargo-make, and our cargo-make scripts use Docker containers, let's use [Docker Prometheus discovery](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#docker_sd_config) instead of static configurations.

Can you please make it work with Fendermint as well, since it's also exporting metrics now? (see https://github.com/consensus-shipyard/ipc/pull/838)

Reply:
We need to make sure all of them live in the same network to make discovery possible. I highly suggest we merge this PR as it is right now to unblock development. What you said can be done as a future task. Do you want me to write a Linear task for that?

Reply:
We discussed this in Slack. The goal of ENG-728 is to set up the observability stack for all processes started by cargo-make. That includes Fendermint and CometBFT, there is no reason to single out the relayer.

In fact, cargo-make does not even have a target to spin up a relayer, so I'm not sure what development this would unblock? This PR will not work as it hardcodes assumptions about the port (9185).

Let's do it the right way since the beginning, @mb1896: with Docker service discovery. What exactly are you concerned about re: Docker network? AFAIK, Docker service discovery scans all networks (and adds the network as a label).